### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jenharris**
I’m a bit confused about where these commits originated (made a bit of a mess over here because of a merge conflict) - please review to make sure nothing appears to be breaking!

### CODE REVIEW

**Comment by jenharris** [comment]
This PR includes all the commits from my previous PR (#179 -it has merge conflicts) so that one can be closed if this one looks ok.

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@
 _site/
 node_modules
 output.log
+
+docs/00-contributing/git-workflow.md
```
Comment (kevwalsh) [inline .gitignore]: @jenharris i don't think we want this in here

File: docs/06-project-management/pm-training.md [hunk 2]
```diff
@@ -94,7 +96,7 @@ Accurately estimating is one of the most difficult things to do. Developers shou
 
 ## <a name="pm-tools"></a>PM Tools
 
-* [Harvest Forecast](../04-how-we-work/tools/harvest-forecast.md)
+* [Resource Guru](../04-how-we-work/tools/harvest-forecast.md)
```
Comment (kevwalsh) [inline docs/06-project-management/pm-training.md]: I don't think this was intentional?

**Review by kevwalsh — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jenharris) [reply .gitignore]: yes, not intentional...

File: docs/06-project-management/pm-training.md (hunk 2 above)
Comment (jenharris) [reply docs/06-project-management/pm-training.md]: nope... thanks

**Review by kevwalsh — APPROVED** [review]

**Pull request merged**